lib/ojs.mli

```diff
@@ -61,6 +61,16 @@ external get: t -> string -> t = "caml_js_get"
 
 external set: t -> string -> t -> unit = "caml_js_set"
 
+val get_with: ('a -> t) -> t -> 'a -> t
```

Comment:
What about exposing directly internal_{get,set}.  It doesn't really lead to simpler code on the call site to pass the "index_to_js" argument than to apply it directly.

Reply:
Yes, let's do that.